utils/amt-integration-tests/tests/features/assignmentdim_view.feature

```diff
@@ -0,0 +1,93 @@
+            | LocalCourseCode   | abc-1053    |
+            | SchoolId          | 53          |
+            | SchoolYear        | 1053        |
+            | SectionIdentifier | si-1053     |
+            | SessionName       | Summer-1053 |
+        And there is a grading period
+            | Descriptor     | First summer session 1053 |
+            | PeriodSequence | 3                         |
+            | SchoolId       | 53                        |
+            | SchoolYear     | 1053                      |
+            | BeginDate      | 1053-07-01 01:23:45       |
+
+
+    Scenario: Ensure the view exists
+        When I query for assignments with identifier "this-row-doesnt-exist"
+        Then it has these columns:
+            | Columns          |
+            | AssignmentKey    |
+            | SchoolKey        |
+            | SourceSystem     |
+            | Title            |
+            | Description      |
+            | StartDateKey     |
+            | EndDateKey       |
+            | DueDateKey       |
+            | MaxPoints        |
+            | SectionKey       |
+            | GradingPeriodKey |
+            | LastModifiedDate |
+
+    Scenario: Happy Path
+        Given there is one Assignment
+            | AssignmentIdentifier | assigndim-happy-path  |
+            | SchoolId             | 53                    |
+            | SourceSystem         | Schoology             |
+            | Title                | A discussion for 1053 |
+            | Description          | The description       |
+            | StartDateTime        | 1053-07-08 09:00      |
+            | EndDateTime          | 1053-07-09 10:00      |
+            | DueDateTime          | 1053-07-09 10:11      |
+            | MaxPoints            | 99                    |
+            | SectionIdentifier    | si-1053               |
+            | LastModifiedDate     | 1053-07-07 09:01      |
+            | AssignmentCategory   | Assignment            |
+        When I query for assignments with identifier "assigndim-happy-path"
+        Then there should be 1 records
+        And the AssignmentDim record should have these values:
+            | AssignmentKey    | assigndim-happy-path                 |
+            | SchoolKey        | 53                                   |
+            | SourceSystem     | Schoology                            |
+            | Title            | A discussion for 1053                |
+            | Description      | The description                      |
+            | StartDateKey     | 1053-07-08                           |
+            | EndDateKey       | 1053-07-09                           |
+            | DueDateKey       | 1053-07-09                           |
+            | MaxPoints        | 99                                   |
+            | SectionKey       | 53-abc-1053-1053-si-1503-Summer-1053 |
+            | GradingPeriodKey | r"[0-9]+-53-1053-07-01"              |
```

Comment:
The first part of the `GradingPeriodKey` is a `GradingPeriodDescriptorId`. Of course we have no way of knowing what that ID will be in advance, so here I used `r"expression"` to signal that a regular expression is in use, and the first component `[0-9]+` signals that we are just expecting a number there. At this point, we're just trusting that the _correct_ number goes into this field. Doing anything else would require substantial effort and probably wouldn't be worth it.